### ISSUE TITLE
PyQt5 example needs an update

### ISSUE THREAD

**Issue by willbelr** [takluyver/pynsist#239]
Hi, thank you very much for your wrapper as once it works, it works great.

However, I spent a day fiddling with the [PyQt5 example](https://github.com/takluyver/pynsist/tree/master/examples/pyqt5). It work as is, but the Python version is quite old so I wanted to max it out for Windows 7 (3.8.7). Turns out it could not be set above 3.7.9, else it requires more recent versions of PyQt5 and sip wheels, which breaks the install and returns a missing DLL error. 

The higher versions I could get with PyQt5 and sip wheels is `Python 3.7.9` with `pyqt5==5.10.1` and `sip==4.19.8`.
In the end, to make it work with Python 3.8.7, those are the wheels I used:
```
pypi_wheels= pyqt5==5.15.6
    pyqt5-qt5==5.15.2
    pyqt5-sip==12.9.0
```

Hence, I think that it would be relevant to update the PyQt5 example with more recent packages versions.

**Comment by takluyver** [comment]
Thanks, updating it sounds like a good idea - do you want to make a pull request?

**Comment by takluyver** [comment]
Fixed by #240.
